### DISCUSSION TITLE
v2.1.1

### DISCUSSION THREAD

**Discussion by matsn0w** [matsn0w/IIS-EmergencyLS-ELS-FiveM#98] [Announcements]
## What's Changed
* fix: set correct dir by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/87
* fix: add check for windows/unix server by @matsn0w in https://github.com/matsn0w/MISS-ELS/pull/95

**Full Changelog**: https://github.com/matsn0w/MISS-ELS/compare/2.1.0...2.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/matsn0w/MISS-ELS/releases/tag/2.1.1'>v2.1.1</a>.</em>

**Comment by SplendidBrit** [comment]
I don't seem to be able to get the lights to sync over the server. other people don't see the same light pattern as I do. Any ideas?

**Reply by matsn0w** [reply]
Hey, that's currently an issue. See #41 for more details.
